src/app/core/store/videos/videos.data.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* tslint:disable */
+import {environment} from '../../../../environments/environment';
+
+export class VideosData {
+  private static readonly VIDEO_HR = {en: 'etoqX2slVEw', cs: ''};
+  private static readonly VIDEO_GOV = {en: '_msiz33O9Tg', cs: ''};
+
+  private static readonly allVideos = {
+    // 'etoqX2slVEw': 20, // HR
```

Comment:
As I am looking at this class more closely, there is no need to have a class here at all since everything is static. And because this is not Java you can define the functions in the file like this:
```typescript
export function getAllVideos() {
  ...
}
```
And constants the way I mentioned in the previous comment. The code would be much cleaner and free of unwanted side-effects.

Reply:
Thanks for the tips. Although, the metric of "cleanness" is subjective and there are no side-effects at the moment anyway ;-)